packages/conjure-client/src/__integTest__/tests.ts

```diff
@@ -75,7 +69,24 @@ describe("Body serde", () => {
     function automaticTest(endpointName: string, index: number, shouldPass: boolean) {
         return async () => {
             if (shouldPass) {
-                return confirmService.confirm(endpointName, index, await (testService as any)[endpointName](index));
+                const endpointResponse = await (testService as any)[endpointName](index);
+                if (endpointName === "receiveBinaryAliasExample") {
```

Comment:
typeof check if endpointResponse is a Blob?

Reply:
typeof doesn't have the fidelity - it just returns `"object"`

Reply:
Ah right sorry maybe `instanceof` would have done the trick... tbh I don't really care for this PR as you're about to change this, but would like the eventual state to ideally not switch on endpoitn names :)